### PR TITLE
🌐🇮🇹 Added Italian language

### PR DESCRIPTION
**Pull request by VitoMars**
I added the Italian language

### CODE REVIEW

**Comment by helmasaur** [comment]
Nice, thanks! I'll look through it 😊.

**Comment by VitoMars** [comment]
Have you looked at it? I'm an Italian native speaker, there shouldn't be any mistakes. I can also take care of other translations if you want

**Comment by helmasaur** [comment]
Sorry, I've been busy this last days. I'm working on it right now 😁

File: locales/it/zodiac.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		"element": 0,
+		"stone": "topazio"
+	},
+	"capricorn": {
+		"name": "Capricorno",
+		"element": 1,
+		"stone": "rubino"
+	},
+	"aquarius": {
+		"name": "Acquario",
```
[the remark this answers is not in the document]
Comment (VitoMars) [reply locales/it/zodiac.json:53]: In Italy the word "Acquario" with the "c" is much more used.

https://youtu.be/8_2bqQjuqEU
This video is extrapolated from an Italian TV channel, probably the most popular channel in Italian television, every year "Paolo Fox" who is the main Italian astrologer, is invited to the most popular broadcasts to talk about the zodiac signs, as you can see from minute 13:00 about, where he talks about the sign of Aquarius the word is written "Acquario" with the "c"

[the remark this answers is not in the document]
Comment (helmasaur) [reply locales/it/zodiac.json:53]: Wow thank you so much for taking the time to explain all this! I'll merge this pull request.

[the remark this answers is not in the document]
Comment (VitoMars) [reply locales/it/zodiac.json:53]: You're welcome, now I try to insert the other translations

**Pull request merged**